parlai/core/torch_classifier_agent.py

```diff
@@ -161,16 +161,16 @@ def __init__(self, opt: Opt, shared=None):
                 )
             if self.use_cuda:
                 self.model.cuda()
+                if self.data_parallel:
+                    self.model = torch.nn.DataParallel(self.model)
+                if self.model_parallel:
```

Comment:
just to make it abundantly clear, might it be good to have it be an `elif` here? since it is _never_ the case that both of these are applied?